cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaSpecCheckerTest.java

```diff
@@ -64,7 +64,7 @@ public void checkEmptyWarnings() {
     @Test
     public void checkKafkaStorage() {
         Kafka kafka = new KafkaBuilder(ResourceUtils.createKafka(NAMESPACE, NAME, 1, IMAGE, HEALTH_DELAY, HEALTH_TIMEOUT,
-            Collections.emptyMap(), Collections.emptyMap(), Collections.emptyMap(),
+            emptyMap(), null, Collections.emptyMap(), Collections.emptyMap(),
```

Comment:
This is weird ... within this file you remove `Collections.` from one `emptyMap` but leave it with the rest of them. Any reason for it? If not, maybe we could keep it consistent at least within the file or within the same line.